### PR TITLE
JS Task 3

### PR DESCRIPTION
**Pull request by KamilWasilonek**
https://kamilwasilonek.github.io/ui-internship-lwo2/html/Corporate/

### CODE REVIEW

**Review by tkachukivan — COMMENTED** [review]
same as in previous pull request, too much hardcoded values which is possible to avoid, plus check functions do too much stuff at once

File: html/Corporate/js/validation/helper-functions.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-useless-escape */
```
Comment (tsvetkovv) [inline html/Corporate/js/validation/helper-functions.js]: where is useless escape? Maybe it's better to put inline exception?

File: html/Corporate/js/validation/contact-validation.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      if (target.tagName === 'INPUT') {
+        let fieldName = target.name;
+        if (fieldName === 'username') {
+          result[0] = checkName(nameInput, errorMessages[0]);
+        } else if (fieldName === 'email') {
+          result[1] = checkEmail(emailInput, errorMessages[1]);
+        } else if (fieldName === 'password') {
+          result[2] = checkPassword(passwordInput, errorMessages[2]);
+        }
+        checkFormStatus(result, submitBtn);
```
Comment (tsvetkovv) [inline html/Corporate/js/validation/contact-validation.js:62]: probably `result` wanna be object like 
```javascript
{
  isNameValid: true,
  isEmailValid: true,
}
```
and it's still easy to traverse through values of object where is needed

**Review by tsvetkovv — COMMENTED** [review]
As already Ivan said
[Avoid side effects](https://github.com/ryanmcdermott/clean-code-javascript#avoid-side-effects-part-1)
[Functions should do one thing](https://github.com/ryanmcdermott/clean-code-javascript#functions-should-do-one-thing)

**Pull request merged**